### ISSUE TITLE
Illegal reflective access

### ISSUE THREAD

**Issue by VMBindraban** [EssentialsX/Essentials#2192]
### Information

Full output of `/ess version`:

```
Server version: 1.12.2-R0.1-SNAPSHOT git-Paper-1562 (MC: 1.12.2)
EssentialsX version: 2.15.0.45
PermissionsEx version: 1.23.4
Vault version: 1.6.7-b${env.TRAVIS_BUILD_NUMBER}
EssentialsXChat version: 2.15.0.45
EssentialsXSpawn version: 2.15.0.45
```

```
Server version: 1.12.2-R0.1-SNAPSHOT git-Spigot-2cf50f0-8166d17 (MC: 1.12.2)
EssentialsX version: 2.15.0.45
PermissionsEx version: 1.23.4
Vault version: 1.6.7-b${env.TRAVIS_BUILD_NUMBER}
EssentialsXChat version: 2.15.0.45
EssentialsXSpawn version: 2.15.0.45
```

Server log (upload `logs/latest.log` to [Gist](https://gist.github.com/)):
https://gist.github.com/VMBindraban/527a6475a79444c923b53fb48f2d23ec

EssentialsX config (upload `plugins/Essentials/config.yml` to [Gist](https://gist.github.com/)): 
https://gist.github.com/VMBindraban/43f3cafdddc76309c6c0a98315404110
### Details

```
WARNING: Illegal reflective access by com.earth2me.essentials.Settings (file:/home/AMP/.ampdata/instances/MCMA/Minecraft/plugins/EssentialsX-2.15.0.45.jar) to field java.lang.reflect.Field.modifiers
```

**Comment by mirolm** [comment]
I receive such warning when i run my server with Java 11. With Java 8 there is no such thing.

```
java version "11" 2018-09-25
Java(TM) SE Runtime Environment 18.9 (build 11+28)
Java HotSpot(TM) 64-Bit Server VM 18.9 (build 11+28, mixed mode)
```
```
Server version: 1.13.1-R0.1-SNAPSHOT git-Spigot-f6a273b-4824655 (MC: 1.13.1)
EssentialsX version: 2.15.0.45
Vault version: 1.7.1-b92
EssentialsXSpawn version: 2.15.0.45
EssentialsXChat version: 2.15.0.45
```
```
WARNING: An illegal reflective access operation has occurred
WARNING: Illegal reflective access by com.earth2me.essentials.Settings (file:/home/ieti/minecraft/plugins/EssentialsX.jar) to field java.lang.reflect.Field.modifiers
WARNING: Please consider reporting this to the maintainers of com.earth2me.essentials.Settings
WARNING: Use --illegal-access=warn to enable warnings of further illegal reflective access operations
WARNING: All illegal access operations will be denied in a future release

```

**Comment by VMBindraban** [comment]
```
user@Ubuntu-1804-bionic-64-minimal ~ # java --version
openjdk 10.0.2 2018-07-17
OpenJDK Runtime Environment (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.2)
OpenJDK 64-Bit Server VM (build 10.0.2+13-Ubuntu-1ubuntu0.18.04.2, mixed mode)
```

**Comment by chrisisonwildcode** [comment]
Came to report same issue

**Comment by mdcfe** [comment]
This was fixed by ba959586e216f3d42888335e39e21b1f4396b652.
